Tests/sdknightly/create_entities_for_nightly_sdk.py

```diff
@@ -30,7 +30,8 @@ def create_incident_field(path: Path, layout_name: str) -> str:
     field.update({
         'name': name,
         'cliName': cliname,
-        'id': f'incident_{cliname}'
+        'id': f'incident_{cliname}',
+        'associatedTypes': [layout_name]
```

Comment:
not sure i understand this, an incident field is associated with incident type, not a layout